internal/pkg/cli/task_run.go

```diff
@@ -1007,6 +1052,7 @@ func BuildTaskRunCmd() *cobra.Command {
 	utilityFlags := pflag.NewFlagSet("Utility", pflag.ContinueOnError)
 	utilityFlags.AddFlag(cmd.Flags().Lookup(followFlag))
 	utilityFlags.AddFlag(cmd.Flags().Lookup(generateCommandFlag))
+	utilityFlags.AddFlag(cmd.Flags().Lookup(acknowledgeSecretsAccessFlag))
```

Comment:
Should this flag be placed under `"Utility"` or `"Task"`? I am not sure 🤔  Open for discussion!